docs/samples/Microsoft.ML.Samples/Dynamic/MatrixFactorization.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Trainers;
+using System;
+using System.Collections.Generic;
+
+// NOTE: WHEN ADDING TO THE FILE, ALWAYS APPEND TO THE END OF IT. 
+// If you change the existinc content, check that the files referencing it in the XML documentation are still correct, as they reference
+// line by line. 
+namespace Microsoft.ML.Samples.Static
+{
+    public partial class TrainerSamples
```

Comment:
>TrainerSamples [](start = 25, length = 14)

in Senja code it's TrainersSamples not TrainerSamples

Reply:
SDCA.cs is used TrainerSamples..

---
In reply to: [229806788](https://github.com/dotnet/machinelearning/pull/1450#discussion_r229806788) [](ancestors = 229806788)

Reply:
Senja says up to me. TrainerSamples (this is also used in Dynamic SDCA'cs) looks better because I don't feel we need two "s" in the name.

---
In reply to: [229807913](https://github.com/dotnet/machinelearning/pull/1450#discussion_r229807913) [](ancestors = 229807913,229806788)